server/package.json

```diff
@@ -9,15 +9,15 @@
   },
   "dependencies": {
     "cookie-parser": "^1.4.5",
-    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "http-errors": "~1.8.0",
     "morgan": "^1.10.0",
     "nodemon": "^2.0.6"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
+    "chai": "^4.3.0",
     "chai-http": "^4.3.0",
+    "dotenv": "^8.2.0",
     "mocha": "^8.2.1"
```

Comment:
I'm not 100% sure, but I think `dotenv` needs to be a dependency, not just a dev dependency.  Also, I'm not seeing `mongoose` included here, did you install it with the `--save` flag?

Reply:
Maybe that was contributing to the issue. Reading about it in a Medium article, it says: 

"If you look carefully, in this scenario, we don’t really need dotenv in production environment at all! So we could have installed it like a dev dependency, right? But you are (probably) requiring dotenv.config() in the files where you are using environment variable, so your app would break if you remove it from the dependencies… " [full article here](https://medium.com/@pdx.lucasm/dotenv-nodemon-a380629e8bff)

I'm removing it from devDependencies and moving it to dependencies.

I'm also adding mongoose. No idea why it didn't show up in the dependencies. Since npm 5.0, the --save flag is the default and doesn't need to be explicit. From the [docs](https://docs.npmjs.com/cli/v6/commands/npm-install):

`npm install` saves any specified packages into `dependencies` by default.
Additionally, you can control where and how they get saved with some
additional flags:
* `-P, --save-prod`: Package will appear in your `dependencies`. **This is the
                     default unless `-D` or `-O` are present.**
* `-D, --save-dev`: Package will appear in your `devDependencies`.
* `-O, --save-optional`: Package will appear in your `optionalDependencies`.
* `--no-save`: Prevents saving to `dependencies`.

Reply:
Good call, I was also confused as you're right, it should have saved the package in dependencies when you installed it 😄